src/_guides/language/effective-dart/usage.md

```diff
@@ -811,27 +813,27 @@ instead of throwing the same exception object using `throw`.
 `rethrow` preserves the original stack trace of the exception. `throw` on the
 other hand resets the stack trace to the last thrown position.
 
-<div class="bad">
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (rethrow)" replace="/throw e/[!$&!]/g"?>
```

Comment:
I think we shouldn't emphasize bad code (because human brains are funny), so let's not highlight "throw e".

Reply:
Highlight removed.